java: exception java.lang.ArrayIndexOutOfBoundsException has already been caught

